### PR TITLE
fix(avatars): small fixes for avatars

### LINKED ISSUE #767: Avatars
## Description

User avatars have a few minor glitches

* [ ] When loading, they flash a couple of times (reproduce: go on the Landing page)
* [ ] The size of the anonymous icon is smaller than the others (42x48 instead of 52x52) making the flashing more evident and misaligning the text (reproduce: look at project pages with at least an anonymous icon -- see image below)
* [ ] When adding a comment to an issue, the user appears to be anonymous (reproduce: add a comment to any existing issue -- see image below)

## Context

Tested on https://dev.renku.ch

***Different icon sizes***
Screenshot_20200217_110634

***Anonymous comment***
Screenshot_20200217_110617

### PR DESCRIPTION
**Pull request by vfried**
Closes #767 

Fixes:
- When loading, they flash a couple of times (reproduce: go on the Landing page)
- The size of the anonymous icon is smaller than the others (42x48 instead of 52x52) making the flashing more evident and misaligning the text (reproduce: look at project pages with at least an anonymous icon)
- When adding a comment to an issue, the user appears to be anonymous (reproduce: add a comment to any existing issue)

### CODE REVIEW

File: src/project/shared/Projects.state.js [hunk 1]
```diff
@@ -42,6 +42,7 @@ class ProjectsCoordinator {
   }
 
   async getFeatured() {
+    if(this.model.get("featured.fetching"))return;
```
Comment (lorenzo-cavazzi) [inline src/project/shared/Projects.state.js]: Does this case happen? If yes, could you add a comment/issue on how to reproduce it? (I would like to fix it)
Anyway, it won't hurt to test if we are not already fetching :+1:

Comment (vfried) [reply src/project/shared/Projects.state.js]: Yes, this happens... i think it was one of the reasons why the avatars where blinking...

in dev:
image

in virginiatest:
image

Comment (lorenzo-cavazzi) [reply src/project/shared/Projects.state.js]: oh wow, indeed it happens (a lot!)
I'll double check the reason

Comment (vfried) [reply src/project/shared/Projects.state.js]: ok, shall i merge this or do you want to investigate?

Comment (lorenzo-cavazzi) [reply src/project/shared/Projects.state.js]: Go on and merge this, your fix works well.
I think fixing the source problem may require a bit more time (or maybe not if it's related to #756) but I wouldn't keep this PR pending in the meanwhile :slightly_smiling_face:

**Pull request merged**